host/tests/L3_TestCases/dsFPD/dsFPDHelperClass.py

```diff
@@ -53,16 +53,17 @@ def __init__(self, testName:str, qcId:str, log:logModule=None ):
 
         # Load test setup configuration
```

Comment:
Need to update test procedure doc for L3 as per latest

Reply:
updated the Path in specification to reflect actual path of the files.